src/filter.rs

```diff
@@ -258,6 +253,64 @@ impl Filter {
                     }
                 }
             }
+
+            if let Some(ref mut tp) = self.tp {
+                assert!(self.true_peak.len() == self.channels as usize);
+                tp.check_true_peak(src, &mut *self.true_peak);
+            }
```

Comment:
Why did you move this after the filter now, and outside the `ftz`?

Reply:
Oh, that's an accident from all the churn. Will move back.

Reply:
No, I just failed reading the diff correctly. This shouldn't matter then